packages/sdk/src/oracle/utils.ts

```diff
@@ -212,25 +212,45 @@ export type RangeState = {
  * @param {Pick} state
  * @returns {RangeState}
  */
-export function rangeStart(state: Pick<RangeState, "startBlock" | "endBlock" | "multiplier">): RangeState {
+export function rangeStart(
```

Comment:
this initialization function is split into 2 blocks, one where a user specifies a max range, the other where no max range is specified and its deteremined by start/end blocks